src/api/window/window.ts

```diff
@@ -5,12 +5,101 @@ import BoundsChangedReply from './bounds-changed';
 import Animation from './animation';
 import { Application } from '../application/application';
 import Transport from '../../transport/transport';
+//import { initialOptions, windowExists, createChildWindow } from '../../environment/openfin-renderer-api';
+//import { windowExists } from '../../environment/openfin-renderer-api';
+import * as _ from 'underscore';
```

Comment:
really don't want underscore for clone and pick.

Reply:
removed underscore